### PR TITLE
PLENTY-158 Add gulp git hook

### PR DESCRIPTION
**Pull request by darius-adeoweb**

### CODE REVIEW

File: .pre-commit [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        exit 0;
+    fi
+
+    # Changes not compiled
+    gulp js
+    gulp sass
+
+    if git diff --name-only --diff-filter=ACM | grep "resources/js/dist\|resources/css"
+    then
+        printf "ERROR: JS/CSS was not compiled before pushing. Run gulp\n"
```
Comment (TheKeymaster) [inline .pre-commit]: I'd like to have `gulp` in quotation marks. This makes it more obvious that this is a command.

Comment (TheKeymaster) [reply .pre-commit]: The moment you've commited, gulp was already executed. You could also just mention that these files need to be added to the current commit in order to be commited.

File: gulpfile.js [hunk 2]
```diff
@@ -68,4 +69,9 @@ gulp.task('sass', function () {
         .pipe(gulp.dest('resources/css'));
 });
 
+gulp.task('install-hooks', function () {
```
Comment (TheKeymaster) [inline gulpfile.js]: Add that this command needs to be executed before developing to the development section in the `README.md`.

Comment (darius-adeoweb) [reply gulpfile.js]: I have added the instructions to the installing section in the README.md now. If development is better, I will update the pull request.

**Pull request merged**